README.md

```diff
@@ -105,39 +121,28 @@ Create the following services:
 * [**Watson Speech To Text**](https://console.bluemix.net/catalog/services/speech-to-text)
 * [**Watson Text To Speech**](https://console.bluemix.net/catalog/services/text-to-speech)
 
-### 3. Import the Conversation workspace
+### 3. Upload the Watson Assistant workspace
 
-Launch the `Watson Assistant` tool. Use the `import` icon button on the right
+Now that our services are created and the app is deployed we need to update tell the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
 
-Find the local version of [`data/assistant/workspaces/voicebot.json`](data/assistant/workspaces/voicebot.json) and select
-`Import`. Find the `Workspace ID` by clicking on the context menu of the new
-workspace and select `View details`. Save this ID for later.
+Each workspace in Watson Assistant has a specific ID, to find the `Workspace ID` for a given workspace, click the context menu of the workspace and select `View details`. The workspace ID can be copied and saved as we'll need it in the next step.
 
-*Optionally*, to view the conversation dialog select the workspace and choose the
-**Dialog** tab, here's a snippet of the dialog:
+Optionally, to view the conversation dialog select the workspace and choose the **Dialog** tab, here's a snippet of the dialog:
 
 ![](doc/source/images/dialog.png)
 
-If running the application using the toolchain then continue back with Step 5 in the previous section [Configuring Environment variables ](#5-configure-environment-variable)
-else move on to Step 4 below.
+### 4. Configure `.env` with credentials
 
-### 4. Configure credentials
-
-The credentials for IBM Cloud services (Assistant, Speech To Text and Text To Speech), can be found in the ``Services`` menu in IBM Cloud,
-by selecting the ``Service Credentials`` option for each service.
-
-The other settings for Conversation was collected during the
-earlier setup step (``WORKSPACE_ID``).
-
-Copy the [`env.sample`](env.sample) to `.env`.
+Our services are created and workspace uploaded. It's now time to let our application run locally and to do we'll configure a simple text file with the values we want to use. We begin by copying the the [`env.sample`](env.sample) file and naming it `.env`.
```

Comment:
let our application run locally and to do that we'll ...

Reply:
done